src/app/containers/ArticleMain/index.jsx

```diff
@@ -22,17 +26,50 @@ const componentsToRender = {
 };
 
 const ArticleMain = ({ articleData }) => {
-  const { content, metadata, promo } = articleData;
-  const { blocks } = content.model;
+  const {
+    brandName,
+    noBylinesPolicy,
+    defaultImage,
+    articleAuthor,
+  } = useContext(ServiceContext);
+
+  const processedData = processArticleData({
+    ...articleData,
+    brandName,
+    noBylinesPolicy,
+    defaultImage,
+  });
 
   return (
     <>
       <ATIAnalytics data={articleData} />
+
       <ChartbeatAnalytics data={articleData} />
-      <MetadataContainer metadata={metadata} promo={promo} />
+
+      <MetadataContainer
+        metadata={path(['metadata'], articleData)}
+        promo={path(['promo'], articleData)}
+        pageSpecificLinkedData={path(
+          ['articleSpecificLinkedData'],
+          processedData,
+        )}
+      />
+
+      <ArticleMetadata
```

Comment:
Can `MetadataContainer` and `ArticleMetadata` be extracted to their own `ArticleMetadata` container, would stop this file from being cluttered

Reply:
Is the clutter you're referring to the props? I think it's good to see exactly what is being passed to components rather than hiding it away somehow. Also I'm not sure how creating another component to wrap `MetadataContainer` and `ArticleMetadata` would solve the clutter because we'd need to pass those props in to then drill them down to the metadata components.

Reply:
I came across a related SO question with an answer that is in favour of explicitness over abstractions https://stackoverflow.com/questions/52621169/react-props-should-i-pass-the-object-or-its-properties-does-it-make-much-diffe